### ISSUE TITLE
20190322 Github 前端趋势榜

### ISSUE THREAD

**Issue by kujian** [kujian/githubTrending#37]
# 20190322 Github 前端趋势榜
[Github 前端开发日报](http://caibaojian.com/c/news)栏目数据来自[Github热榜](http://news.caibaojian.com/)（我开发的爬虫），每日分享HTML、CSS、JavaScript、Vue等开源项目，为开发者提供动力，点击Star按钮来关注这个项目，点击Watch来收听每日的更新[Github主页](https://github.com/kujian/githubTrending)
## JavaScript
* [clone95/Machine-Learning-Study-Path-March-2019](https://github.com/clone95/Machine-Learning-Study-Path-March-2019) （A complete ML study path, focused on TensorFlow and Scikit-Learn）
* [vuejs/vue](https://github.com/vuejs/vue) （        A progressive, incrementally-adoptable JavaScript framework for building UI on the web.      ）
* [scutan90/DeepLearning-500-questions](https://github.com/scutan90/DeepLearning-500-questions) （深度学习500问，以问答形式对常用的概率知识、线性代数、机器学习、深度学习、计算机视觉等热点问题进行阐述，以帮助自己及有需要的读者。 全书分为15个章节，近20万字。由于水平有限，书中不妥之处恳请广大读者批评指正。 未完待续............ 如有意合作，联系scutjy2015@163.com 版权所有，违权必究 Tan 2018.06）
* [alexfoxy/laxxx](https://github.com/alexfoxy/laxxx) （Simple &amp; light weight vanilla javascript plugin to create beautiful animations things when you scrolllll!! Harness the power of the most intuitive interaction and make your websites come alive!）
* [syhyz1990/baiduyun](https://github.com/syhyz1990/baiduyun) （油猴脚本 直接下载百度网盘和百度网盘分享的文件,直链下载超级加速）
* [CyC2018/CS-Notes](https://github.com/CyC2018/CS-Notes) （? Computer Science Learning Notes）
* [lyricat/wechat-format](https://github.com/lyricat/wechat-format) （微信公众号排版编辑器，Markdown 到给微信特制的 HTML）
* [ammonb/stereogram-raycaster](https://github.com/ammonb/stereogram-raycaster) （A raycaster that renders to random dot stereogram）
* [facebook/react](https://github.com/facebook/react) （A declarative, efficient, and flexible JavaScript library for building user interfaces.）
* [nytimes/library](https://github.com/nytimes/library) （A collaborative documentation site, powered by Google Docs.）
* [vvo/gifify](https://github.com/vvo/gifify) （&#x1f63b; Convert any video file to an optimized animated GIF.）
* [mgechev/mk.js](https://github.com/mgechev/mk.js) （Canvas implementation of basic fighting game which allows multiplayer game over the network.）
* [30-seconds/30-seconds-of-code](https://github.com/30-seconds/30-seconds-of-code) （Curated collection of useful Javascript snippets that you can understand in 30 seconds or less.）
* [kevinwolfcr/formal](https://github.com/kevinwolfcr/formal) （&#x1f454; Elegant cross-platform form management primitives for the react hooks era.）
* [mozilla/send](https://github.com/mozilla/send) （File Sharing Experiment）
* [axios/axios](https://github.com/axios/axios) （Promise based HTTP client for the browser and node.js）
* [trekhleb/javascript-algorithms](https://github.com/trekhleb/javascript-algorithms) （Algorithms and data structures implemented in JavaScript with explanations and links to further readings）
* [francisrstokes/hexnut](https://github.com/francisrstokes/hexnut) （&#x1f529; Hexnut is a middleware based, express/koa like framework for web sockets）
* [agalwood/Motrix](https://github.com/agalwood/Motrix) （A full-featured download manger.）
* [facebook/create-react-app](https://github.com/facebook/create-react-app) （Create React apps with no build configuration.）
* [nosir/cleave.js](https://github.com/nosir/cleave.js) （Format input text content when you are typing...）
* [Webiny/webiny-js](https://github.com/Webiny/webiny-js) （Serverless CMS (GraphQL + React)）
* [GoogleChrome/puppeteer](https://github.com/GoogleChrome/puppeteer) （Headless Chrome Node API）
* [airbnb/javascript](https://github.com/airbnb/javascript) （        JavaScript Style Guide      ）
* [mrdoob/three.js](https://github.com/mrdoob/three.js) （        JavaScript 3D library.      ）
* [NervJS/taro](https://github.com/NervJS/taro) （京东多端统一开发框架，支持用 React 的开发方式编写一次代码，生成能运行在微信小程序、H5、React Native 等的应用。）
* [facebook/react-native](https://github.com/facebook/react) （        A framework for building native apps with React.      ）
* [iview/iview-admin](https://github.com/iview/iview-admin) （Vue 2.0 admin management system template based on iView）
* [WebReflection/hyperHTML](https://github.com/WebReflection/hyperHTML) （        A Fast &amp; Light Virtual DOM Alternative      ）
* [Molunerfinn/PicGo](https://github.com/Molunerfinn/PicGo) （A simple &amp; beautiful tool for pictures uploading built by electron-vue）
* [alibaba/weex-ui](https://github.com/alibaba/weex-ui) （? A rich interaction, lightweight, high performance UI library based on Weex.）
* [maomao1996/Vue-mmPlayer](https://github.com/maomao1996/Vue-mmPlayer) （基于Vue2的在线音乐播放器（PC） Online music player （vue-music））

## Vue
* [dcloudio/uni-app](https://github.com/dcloudio/uni-app) （使用 Vue.js 开发跨平台应用的前端框架）
* [PanJiaChen/vue-element-admin](https://github.com/PanJiaChen/vue-element-admin) （        vue2 admin / a management system template      ）
* [ElemeFE/element](https://github.com/ElemeFE/element) （        A Vue.js 2.0 UI Toolkit for Web      ）
* [iview/iview](https://github.com/iview/iview) （        A high quality UI Toolkit with both Vue.js 2.0 and 1.0      ）
* [vueComponent/ant-design-vue](https://github.com/vueComponent/ant-design-vue) （An enterprise-class UI components based on Ant Design and Vue. ?）
* [macrozheng/mall-admin-web](https://github.com/macrozheng/mall-admin-web) （mall-admin-web是一个电商后台管理系统的前端项目，基于Vue+Element实现。 主要包括商品管理、订单管理、会员管理、促销管理、运营管理、内容管理、统计报表、财务管理、权限管理、设置等功能。）
* [heyui/heyui](https://github.com/heyui/heyui) （UI Toolkit for Web, Vue2.0）
* [liuzhuoling2011/baidupcs-web](https://github.com/liuzhuoling2011/baidupcs-web) （基于vue2+nuxt构建的高仿饿了么（2018版））
* [geektime-geekbang/geektime-vue-1](https://github.com/geektime-geekbang/geektime-vue-1) （极客时间基础篇代码）
* [Armour/vue-typescript-admin-template](https://github.com/Armour/vue-typescript-admin-template) （&#x1f596; A vue + typescript minimal admin template）
* [buefy/buefy](https://github.com/buefy/buefy) （Lightweight UI components for Vue.js based on Bulma）
* [syuilo/misskey](https://github.com/syuilo/misskey) （A Twitter-like SNS ✨??✨）
* [salomonelli/best-resume-ever](https://github.com/salomonelli/best-resume-ever) （? ? Build fast ? and easy multiple beautiful resumes and create your best CV ever! Made with Vue and LESS.）
* [mizuka-wu/vue2-verify](https://github.com/mizuka-wu/vue2-verify) （vue的验证码插件）
* [airyland/vux](https://github.com/airyland/vux) （        Mobile web UI Components based on Vue and WeUI (Vue@2.x)      ）
* [didi/mand-mobile](https://github.com/didi/mand-mobile) （? A mobile UI toolkit, based on Vue.js 2, designed for financial scenes.）
* [nono1526/vue-material-year-calendar](https://github.com/nono1526/vue-material-year-calendar) （A full 12-Month view calendar made by vue.js.）
* [Akryum/vue-virtual-scroller](https://github.com/Akryum/vue-virtual-scroller) （Smooth scroll with any amount of data）

## HTML
* [Alivon/Panda-Learning](https://github.com/Alivon/Panda-Learning) （熊猫学习Panda-Learning是一个辅助学习强国的软体,帮助挤不出时间，却仍然需要学习的积极分子，盼达学习强国(xuexiqiangguo)）
* [hamukazu/lets-get-arrested](https://github.com/hamukazu/lets-get-arrested) （This project is intended to protest against the police in Japan）
* [changtuiqie/WeChatAgreement.WebApi.Simple](https://github.com/changtuiqie/WeChatAgreement.WebApi.Simple) （mac微信,ipad微信 webapi封装好的实现方案，免IIS一键部署。 可实现微信80%功能；支持62数据登录、扫码登录、收发朋友圈、查看朋友圈、微信建群、微信拉人进群、微信公众号阅读、微信消息收发、微信附近的人定位、微信添加好友、微信红包接收、微信防撤回、分享小程序、微信加粉、微信收藏、微信标签等）
* [ColorlibHQ/AdminLTE](https://github.com/ColorlibHQ/AdminLTE) （AdminLTE - Free Premium Admin control Panel Theme Based On Bootstrap 3.x）
* [ghosh/micromodal](https://github.com/ghosh/micromodal) （⭕ Tiny javascript library for creating accessible modal dialogs）
* [iliakan/javascript-tutorial-en](https://github.com/iliakan/javascript-tutorial-en) （        Modern JavaScript Tutorial       ）
* [google/styleguide](https://github.com/google/styleguide) （        Style guides for Google-originated open-source projects      ）
* [facebookresearch/fastText](https://github.com/facebookresearch/fastText) （Library for fast text representation and classification.）
* [kennethreitz/requests-html](https://github.com/kennethreitz/requests-html) （Pythonic HTML Parsing for Humans™）
* [tc39/ecma262](https://github.com/tc39/ecma262) （        Status, process, and documents for ECMA262      ）
* [kujian/frontendDaily](https://github.com/kujian/frontendDaily) （前端开发技术日报，每日分享互联网最精彩的前端技术、前端资讯、后端编程、设计和资源等，欢迎关注Watch）
* [greggman/webgl-fundamentals](https://github.com/greggman/webgl-fundamentals) （WebGL lessons that start with the basics）
* [portainer/portainer](https://github.com/portainer/portainer) （        Simple management UI for Docker      ）
* [QSCTech/zju-icicles](https://github.com/QSCTech/zju-icicles) （浙江大学课程攻略共享计划）
* [BlackrockDigital/startbootstrap-sb-admin-2](https://github.com/BlackrockDigital/startbootstrap-sb-admin-2) （        A free, open source, Bootstrap admin theme created by Start Bootstrap      ）
* [uikit/uikit](https://github.com/uikit/uikit) （A lightweight and modular front-end framework for developing fast and powerful web interfaces）
* [xitu/javascript-tutorial-zh](https://github.com/xitu/javascript-tutorial-zh) （Modern JavaScript Tutorial）
* [puikinsh/Adminator-admin-dashboard](https://github.com/puikinsh/Adminator-admin-dashboard) （Adminator is a easy to use and well design admin dashboard template for web apps, websites, services and more）
* [flutterchina/flutter-in-action](https://github.com/flutterchina/flutter-in-action) （《Flutter实战》电子书）
* [climanifeste/climanifeste](https://github.com/climanifeste/climanifeste) （Manifeste écologique des professionnels de l'informatique）

## CSS
* [twbs/bootstrap](https://github.com/twbs/bootstrap) （The most popular HTML, CSS, and JavaScript framework for developing responsive, mobile first projects on the web.      ）
* [github/personal-website](https://github.com/github/personal-website) （Code that'll help you kickstart a personal website that showcases your work as a software developer.）
* [tailwindcss/tailwindcss](https://github.com/tailwindcss/tailwindcss) （A utility-first CSS framework for rapid UI development.）
* [jgthms/bulma](https://github.com/jgthms/bulma) （        Modern CSS framework based on Flexbox      ）
* [carloscuesta/gitmoji](https://github.com/carloscuesta/gitmoji) （        An emoji guide for your commit messages. ?       ）
* [Autodesk/standard-surface](https://github.com/Autodesk/standard-surface) （White paper describing Arnold's standard surface shader.）
* [necolas/normalize.css](https://github.com/necolas/normalize.css) （A collection of HTML element and attribute style-normalizations      ）
* [nostalgic-css/NES.css](https://github.com/nostalgic-css/NES.css) （NES-style CSS Framework | ファミコン風CSSフレームワーク）
* [yueshutong/JavaMonitor](https://github.com/yueshutong/JavaMonitor) （一款适用于Java应用的在线性能监控系统（WY））
* [LiangJunrong/document-library](https://github.com/LiangJunrong/document-library) （jsliang 的文档库. 里面包含了所有的前端文章，例如 vue、react,、angular、微信小程序、设计模式等……）
* [theme-next/hexo-theme-next](https://github.com/theme-next/hexo-theme-next) （Elegant and powerful theme for Hexo.）
* [rstacruz/cheatsheets](https://github.com/rstacruz/cheatsheets) （My cheatsheets）
* [mmistakes/minimal-mistakes](https://github.com/mmistakes/minimal-mistakes) （        ? A flexible two-column Jekyll theme. Perfect for personal sites, blogs, and portfolios hosted on GitHub or your own server.      ）
* [Microsoft/TypeScript-Node-Starter](https://github.com/Microsoft/TypeScript-Node-Starter) （        A starter template for TypeScript and Node with a detailed README describing how to use the two together.      ）
* [Requarks/wiki](https://github.com/Requarks/wiki) （Wiki.js | A modern, lightweight and powerful wiki app built on NodeJS, Git and Markdown）
* [bcye/Hello-Goodbye](https://github.com/bcye/Hello-Goodbye) （A Chrome extension to block annoying chat bubbles from Intercom &amp; Co.）
* [StylishThemes/GitHub-Dark](https://github.com/StylishThemes/GitHub-Dark) （         Dark GitHub style      ）
* [handong1587/handong1587.github.io](https://github.com/handong1587/handong1587.github.io) （        A Jekyll-based theme designed for documentation and help systems. See the link for detailed instructions on setting up and configuring everything.      ）

## Unknow
* [h5bp/Front-end-Developer-Interview-Questions](https://github.com/h5bp/Front-end-Developer-Interview-Questions) （        A list of helpful front-end related questions you can use to interview potential candidates, test yourself or completely ignore.      ）

## 关于我
 [前端开发博客](http://caibaojian.com/) 站长，[微博](https://weibo.com/kujian) 粉丝18万，[微信公众号](http://t.cn/EV7pcM1)。
如果你觉得我的分享对你有帮助，不妨考虑打赏我[一杯咖啡](http://caibaojian.com/donation)，以资鼓励。
![打赏](https://upload-images.jianshu.io/upload_images/570843-db4053c67a8c9ea9.png)
## 推荐
1. [掘金小册8折优惠，还有全网优惠教程](http://caibaojian.com/goods)
2. [36个小而美的赚钱项目，让你月入10万](http://t.cn/EV7YgYw)
## 微信群
加入微信群，每天晚上我会汇总一天最好的文章给你。
![qianduanquan](https://user-images.githubusercontent.com/3055447/38468989-651132ac-3b80-11e8-8e6b-15122322a9d7.png)
